### PR TITLE
ACCESS-Hub name change

### LINKED ISSUE #28: Update references to ACCESS-Hub
For consistency with the organisational rename, and other repos, this repository was changed from ACCESSHub to ACCESS-Hub. References in the docs need to be changed to reflect this.

### PR DESCRIPTION
**Pull request by aidanheerdegen**
Fixes #28

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-site_name: ACCESSHub Demo
+site_name: ACCESS-Hub
 
 # Git repository
-repo_url: https://github.com/ACCESS-NRI-Community/ACCESSHub
-repo_name: ACCESSHub
+repo_url: https://github.com/ACCESS-NRI-Community/ACCESS-Hub
```
Comment (ccarouge) [inline mkdocs.yml:4]: Need to change the path from ACCESS-NRI-Community to ACCESS-Hub:
https://github.com/ACCESS-Hub/ACCESS-Hub

File: docs/Help/modify_documentation.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ There are two ways to modify the documentation:
 
 ## Open an issue
 
-For all additions or modifications to the ACCESSHub site, it is recommended to start by opening an [Issue](https://github.com/ACCESS-NRI-Community/ACCESSHub/issues) in the ACCESSHub GitHub repository. After creation, please [assign the Issue to yourself in the right-hand side panel](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users) if you intend on working on the issue.
+For all additions or modifications to the ACCESS-Hub site, it is recommended to start by opening an [Issue](https://github.com/ACCESS-NRI-Community/ACCESS-Hub/issues) in the ACCESS-Hub GitHub repository. After creation, please [assign the Issue to yourself in the right-hand side panel](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users) if you intend on working on the issue.
```
Comment (ccarouge) [inline docs/Help/modify_documentation.md:10]: Wrong URL, still refers to ACCESS-NRI-Community. Correct URL: https://github.com/ACCESS-Hub/ACCESS-Hub/issues

**Review by ccarouge — CHANGES_REQUESTED** [review]

File: docs/Help/modify_documentation.md (hunk 2 above)
Comment (aidanheerdegen) [reply docs/Help/modify_documentation.md:10]: Thanks.

File: mkdocs.yml (hunk 1 above)
Comment (aidanheerdegen) [reply mkdocs.yml:4]: Thanks

**Review by ccarouge — APPROVED** [review]

**Pull request merged**